src/site/docs/tutorials/get-started/index.markdown

```diff
@@ -397,79 +392,79 @@ Dart Editor opens a new panel,
 called the **Output view**,
 and displays the output of the helloworld app.
 
-<img class="scale-img-max" src="images/helloworld-output.png"
-     alt="Dart Editor with helloworld output view"/>
-
 ##Create a web app {#create-web-app}
 
 Now let's create a web application.
 As you did when creating a command-line application,
 click the **New Application** button
 <img class="scale-img-max" src="images/newapp.png" width="17" height="16" alt="New App button"/>.
-Name the application `clickme`.
-This time you are creating a web app,
-so select **Web application** from the list
-in the New Application dialog window:
 
-<img class="scale-img-max" src="images/new-click-me.png"
-     alt="Create a new web application"/>
+<ol>
+<li markdown="1">
+Type `simple` in the application name text field.
+</li>
 
-Dart Editor creates the directory and files needed
-for a boilerplate web application.
+<li markdown="1">
+Type or browse to the directory where you want to save the files.
+</li>
 
-<img class="scale-img-max" src="images/click-me-files.png"
-     alt="Dart Editor with new web application files"/>
+<li markdown="1">
+Select **Generate sample content**.
+</li>
+
+<li markdown="1">
+Select **Uber Simple Web Application** from the list.
+</li>
+
+<li markdown="1">
+Click **Finish**.
+</li>
+</ol>
+
+Dart Editor creates the directory and files needed for the most
+basic boilerplate web application which prints "Your Dart app is running"
+in the browser window.
 
 As before, the directory is named after your application.
-So is the Dart source file that contains the main() function.
-In addition, Dart Editor creates an HTML file that hosts the app.
+So is the Dart source file that contains the main() function is
+called main.dart.
+In addition, Dart Editor creates an HTML file, called index.html,
```

Comment:
too many commas! How about:

In addition, Dart Editor creates `web/index.html`, an HTML file that hosts the app.